coco/worker.py

```diff
@@ -40,6 +40,16 @@ def signal_handler(sig, frame):
 signal.signal(signal.SIGINT, signal_handler)
 
 
+async def _open_redis_connection():
```

Comment:
Oh great. I was going to suggest pulling out the connection parameters somewhere, but I didn't want to expand the scope too much. And this is even better than that!